datahub-web-react/src/utils/customGraphQL/useGetEntitySearchResults.ts

```diff
@@ -0,0 +1,25 @@
+import { EntityType, SearchInput } from '../../types.generated';
+import { useGetSearchResultsQuery } from '../../graphql/search.generated';
+
+type AllEntityInput<T, K> = Pick<T, Exclude<keyof T, keyof K>> & K;
+
+export function useGetEntitySearchResults(
+    input: AllEntityInput<SearchInput, { type?: EntityType }>,
+    searchTypes: Array<EntityType>,
+) {
+    const result: any = {};
```

Comment:
While you're here- can you fix the type of the result? The lack of type caused issues in the past: https://github.com/linkedin/datahub/pull/2553/files.